### ISSUE TITLE
gazebo:unknown macro name: xacro:base None（as_arm_bringup.launch）

### ISSUE THREAD

**Issue by ChenyangRan** [yao62995/AS_6Dof_Arm#8]
你好，我在运行roslaunch as_arm_gazebo as_arm_bringup.launch时候报错unknown macro name: xacro:base None None
![Screenshot from 2019-05-24 23-36-09](https://user-images.githubusercontent.com/29306735/58339710-c5239a00-7e7c-11e9-9993-7c5feac69b5e.png)
从报错上看应是load模型时候出了问题，采用的是原程序的command="$(find xacro)/xacro.py '$(find as_arm_description)/urdf/as_arm.xacro'"
请问应该怎么解决？
配置：
Ubuntu16.04
ros-kinetic
Gazebo 7.15.0

**Comment by TONG369** [comment]
> 你好，我在运行roslaunch as_arm_gazebo as_arm_bringup.launch时候报错unknown macro name: xacro:base None None
> 
> 从报错上看应是load模型时候出了问题，采用的是原程序的command="$(find xacro)/xacro.py '$(find as_arm_description)/urdf/as_arm.xacro'"
> 请问应该怎么解决？
> 配置：
> Ubuntu16.04
> ros-kinetic
> Gazebo 7.15.0

您好，请问您的问题解决了吗？我遇到了和您一样的问题？

**Comment by ChenyangRan** [comment]
> > 你好，我在运行roslaunch as_arm_gazebo as_arm_bringup.launch时候报错unknown macro name: xacro:base None None
> > 从报错上看应是load模型时候出了问题，采用的是原程序的command="$(find xacro)/xacro.py '$(find as_arm_description)/urdf/as_arm.xacro'"
> > 请问应该怎么解决？
> > 配置：
> > Ubuntu16.04
> > ros-kinetic
> > Gazebo 7.15.0
> 
> 您好，请问您的问题解决了吗？我遇到了和您一样的问题？

No, and because the moviet_simple_grasp can't be used in Ubuntu16.04.
Then I don't use it anymore.

**Comment by TONG369** [comment]
thanks your reply,thank you very much 





在 2019-10-18 14:42:08，"ChenyRan" <notifications@github.com> 写道：


你好，我在运行roslaunch as_arm_gazebo as_arm_bringup.launch时候报错unknown macro name: xacro:base None None
从报错上看应是load模型时候出了问题，采用的是原程序的command="$(find xacro)/xacro.py '$(find as_arm_description)/urdf/as_arm.xacro'"
请问应该怎么解决？
配置：
Ubuntu16.04
ros-kinetic
Gazebo 7.15.0

您好，请问您的问题解决了吗？我遇到了和您一样的问题？

No, and because the moviet_simple_grasp can't be used in Ubuntu16.04.
Then I don't use it anymore.

**Comment by 7Savage** [comment]
https://answers.ros.org/question/242044/book-mastering-ros-for-robotics-programming-2nd-edn-chapter-2-working-with-3d-robot-modeling-in-ros/
